attribution-graphs-frontend/feature_examples/init-feature-examples.js

```
@@ -0,0 +1,60 @@
+      renderAll.feature()
+      if (visState.hideStaleOutputs) sel.classed('is-stale-output', 0)
+    }
+
+    return feature
+  }
+
+  async function loadFeature(scan, featureIndex){
+    try {
+      var feature = await  util.getFile(`/features/${scan}/${featureIndex}.json`)
+    } catch {
+      var feature = {isDead: true, statistics: {}}
```

Biome (1.9.4)

[error] Shouldn't redeclare 'feature'. Consider to delete it or rename it.

'feature' is defined here:


(lint/suspicious/noRedeclare)